tests/test_feedexport.py

```diff
@@ -622,6 +622,45 @@ def test_export_dicts(self):
         yield self.assertExportedCsv(items, ['egg', 'foo'], rows_csv, ordered=False)
         yield self.assertExportedJsonLines(items, rows_jl)
 
+    @defer.inlineCallbacks
+    def test_export_dicts_warning(self):
```

Comment:
This looks great, but could you also add tests using the `Item` class? Thanks!

Reply:
Added a new test case! Thanks!